### ISSUE TITLE
Importing videos longer than 5 minutes will bring down the entire Win10

### ISSUE THREAD

**Issue by SiuLeeBoss** [Scholar01/sd-webui-mov2mov#123]
When I import a video that is more than 5 minutes long, it can't read the fps, and then the whole win10 is unresponsive, and the whole interface is stuck and can't move.

**Comment by 00RohitRoshan** [comment]
Same for windows11

**Comment by optera15** [comment]
Yes, there is a massive memory leak. I can watch it leak frame by frame till my GPU explodes and Wndows11 dies. This should be super easy to find and fix. Just stop leaking every frame, please!

**Comment by lktinh2018** [comment]
Just killed :(

**Comment by bob200001** [comment]
Experiencing same issue. Any video over 5 min can't process/import fps and my 32 gigs of ram is maxed out on my system and crashes/locks the pc.
